Justfile

```
@@ -36,15 +36,16 @@ _parse_combos:
 # parse build.yaml and filter targets by expression
 _parse_targets $expr:
     #!/usr/bin/env bash
-    attrs="[.board, .shield, .snippet]"
+    attrs="[.board, .shield, .snippet, .\"artifact-name\"]"
     filter="(($attrs | map(. // [.]) | combinations), ((.include // {})[] | $attrs)) | join(\",\")"
     echo "$(yq -r "$filter" build.yaml | grep -v "^," | grep -i "${expr/#all/.*}")"
 
+
 # build firmware for single board & shield combination
 _build_single $board $shield $snippet *west_args:
```

```suggestion
_build_single $board $shield $snippet $artifact *west_args:
```
I don't think this works without passing the extra argument